proton-c/src/tests/fuzz/fuzz-sniff-header.c

```diff
@@ -0,0 +1,15 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "core/autodetect.h"
+
+#ifdef __cplusplus
```

Comment:
The declaration only has to be extern "C" so you could put:
    #ifdef __cplusplus
    extern "C"
    #endif
    int LLVMFuzzerTestOneInput(uint8_t *data, size_t size);
In a header file. That is sufficient to give the function the correct linkage. In fact I'm surprised there isn't already a declaration like this in libfuzzer.